### PR TITLE
Use account boc

### PR DESCRIPTION
**Pull request by melsomino**

### CODE REVIEW

File: ton_client/client/tonclient.h [hunk 1]
```diff
@@ -0,0 +1,51 @@
+#pragma once
```
Comment (AlexeyVavilin) [inline ton_client/client/tonclient.h:1]: Why this file is added in the PR?

File: ton_client/client/src/tvm/mod.rs [hunk 2]
```diff
@@ -18,10 +18,12 @@ use ton_sdk::Contract;
 use crate::client::ClientContext;
 use crate::types::{ApiResult, ApiError};
 use crate::dispatch::DispatchTable;
+use ton_block::MsgAddressInt;
 
 #[derive(Serialize, Deserialize)]
 #[allow(non_snake_case)]
 pub(crate) struct ParamsOfLocalRunGet {
+    pub bocBase64: Option<String>,
```
Comment (AlexeyVavilin) [inline ton_client/client/src/tvm/mod.rs:26]: Maybe better pass struct `Contract` rather than separate fields `bocBase64`, `codeBase64`, `dataBase64`, `balance` and `last_paid`?

File: ton_client/client/src/contracts/run.rs [hunk 3]
```diff
@@ -305,13 +305,22 @@ pub(crate) fn local_run(context: &mut ClientContext, params: ParamsOfLocalRun) -
 
     let key_pair = params.key_pair.map(|pair| pair.decode()).transpose()?;
 
-    let account = params.account.as_ref().map(|acc| 
+    let account = params.account
```
Comment (AlexeyVavilin) [inline ton_client/client/src/contracts/run.rs:308]: DO NOT use formatter in functional PRs. If you want to format code acording to some scheme you should make separate PR for that

File: ton_client/client/tonclient.h (hunk 1 above)
Comment (melsomino) [reply ton_client/client/tonclient.h:1]: It is a common c/c++ header for all bindings that will link a core library.

File: ton_client/client/src/tvm/mod.rs (hunk 2 above)
Comment (melsomino) [reply ton_client/client/src/tvm/mod.rs:26]: The next release may be

File: ton_client/client/src/contracts/run.rs (hunk 3 above)
Comment (melsomino) [reply ton_client/client/src/contracts/run.rs:308]: ok
